docs/management/admin/bidirectional-actions.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[bidirectional-actions]]
+= Bidirectional response actions
+
+:frontmatter-description: Perform response actions on hosts protected by third-party endpoint security systems.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [reference]
+:frontmatter-tags-user-goals: [manage]
+
+preview::[]
+
+[discrete]
+[[sentinelone-response-actions]]
+== SentinelOne response actions
+
+SentinelOne response actions allow you to direct SentinelOne to perform actions on protected hosts without needing to leave the {elastic-sec} UI. Prior <<response-actions-config,configuration>> is required to connect {elastic-sec} with SentinelOne.
```

Comment:
Excellent point, thanks! I like the slightly more wordy first version, because it seems relevant that SentinelOne is still actually performing these actions, we're just telling S1 to do it. Slight tweak to make this a complete sentence:
```suggestion
You can direct SentinelOne to perform response actions on protected hosts without leaving the {elastic-sec} UI. Prior <<response-actions-config,configuration>> is required to connect {elastic-sec} with SentinelOne.
```